app/license.go

```diff
@@ -331,3 +331,13 @@ func (s *Server) renewalTokenValid(tokenString, signingKey string) (bool, error)
 	}
 	return true, nil
 }
+
+// GenerateLicenseRenewalEmail returns a link that points to the CWS where clients can renew license
+func (s *Server) GenerateLicenseRenewalEmail() (string, *model.AppError) {
```

Comment:
Given that this generates just the renewal link, I suggest we can name as:

```suggestion
func (s *Server) GenerateLicenseRenewalLink() (string, *model.AppError) {
```